lib/widgets/forms/task_form.dart

```diff
@@ -289,11 +308,78 @@ class _TaskFormState extends State<TaskForm> {
 				onChanged: (val) => setState(() {
 					isDataChanged = task.description != val;
 					task.description = val;
-				})
+				}),
 			)
 		);
 	}
 
+	Widget buildSubtasksFields() {
+		return Column(
+		  children: [
+				ImplicitlyAnimatedReorderableList<MapEntry<Key, String>>(
```

Comment:
Is that function copied from the task list?

Reply:
Yes, it is, I don't know how to parametrize this

Reply:
Here you go :slightly_smiling_face: